static/app/views/discover/table/quickContext/issueContext.spec.tsx

```diff
@@ -133,44 +133,105 @@ describe('Quick Context Content Issue Column', function () {
+            {
+              author: {name: 'Maisey the Dog', id: '1231'},
+              commits: [
+                {
+                  message: 'feat(simulator): Add option for multiple squirrels',
+                  id: 'ab2709293d0c9000829084ac7b1c9221fb18437c',
+                  dateCreated: '2012-09-08T04:15:12',
+                },
+              ],
+            },
+          ],
+        },
+      });
+      renderIssueContext();
+
+      // Make sure the title renders in the singular, since there's only one commit
+      expect(await screen.findByText(/Suspect Commit/i)).toBeInTheDocument();
+      expect(screen.queryByText(/Suspect Commits/i)).not.toBeInTheDocument();
+
+      // Ensure all commit data is present
+      expect(screen.getByText(/MD/i)).toBeInTheDocument();
+      expect(screen.getByText(/View commit/i)).toBeInTheDocument();
+      expect(screen.getByText(/by/i)).toBeInTheDocument();
+      expect(screen.getByText(/Maisey the Dog/i)).toBeInTheDocument();
     });
 
-    MockApiClient.addMockResponse({
-      method: 'GET',
-      url: '/projects/org-slug/cool-team/events/6b43e285de834ec5b5fe30d62d549b20/committers/',
-      body: {
-        committers: [
-          {
-            author: {name: 'Max Bittker', id: '1'},
-            commits: [
-              {
-                message: 'feat: Added new feature',
-                score: 4,
-                id: 'ab2709293d0c9000829084ac7b1c9221fb18437c',
-                repository: TestStubs.Repository(),
-                dateCreated: '2018-03-02T18:30:26Z',
-                pullRequest: {
-                  externalUrl: 'url',
+    it('Renders multiple suspect commits', async () => {
+      MockApiClient.addMockResponse({
+        url: '/issues/3512441874/events/oldest/',
+        method: 'GET',
+        body: {
+          eventID: '6b43e285de834ec5b5fe30d62d549b20',
+        },
+      });
+
+      MockApiClient.addMockResponse({
+        method: 'GET',
+        url: '/projects/org-slug/cool-team/events/6b43e285de834ec5b5fe30d62d549b20/committers/',
+        body: {
+          committers: [
+            {
+              author: {name: 'Maisey the Dog', id: '1231'},
+              commits: [
+                {
+                  message: 'feat(simulator): Add option for multiple squirrels (#1121)',
+                  id: 'ab2709293d0c9000829084ac7b1c9221fb18437c',
+                  dateCreated: '2012-09-08T04:15:12',
+                },
+              ],
+            },
+            {
+              author: {name: 'Charlie Bear', id: '1121'},
+              commits: [
+                {
+                  message:
+                    'ref(simulator): Split leaderboard calculations into separate functions (#1231)',
+                  id: 'fe29668b24cea6faad8afb8f6d9417f402ef9c18',
+                  dateCreated: '2012-04-15T09:09:12',
                 },
-              },
-            ],
-          },
-        ],
-      },
+              ],
+            },
```

Comment:
you might just define the two commits at the top of this document and reuse them